tests/unit/io/TBOutputFormat_unittest.cc

```diff
@@ -154,20 +155,20 @@ TEST(TBOutputFormatTest, fail_convert_scalar)
 	test_case_scalar_error<complex128_t>(1);
 };
 
-TYPED_TEST(TBOutputFormatTest, convert_all_types_histo)
+TYPED_TEST(TBOutputFormatTest, DISABLED_convert_all_types_histo)
```

Comment:
These tests are disabled because the parameter framework does not support the `std::vector` type.

Reply:
In what sense? 
Actually there is a pr by Sergey that Viktor said he wants to merge so that should allow vectors

Reply:
Same here, the `Parameter` class does not have any `add` method to add `std::vector` objects. Moreover, `Any` still not support `std::vector` (however, this should be fixed by Viktor's PR).

Reply:
Not quite (the pr disables testing so I’m not sure what will happen with it) . But what you could do is to cherry pick Sergey’s commits that add std vector support in your pr here.

Reply:
Oh and then also here: don’t use sgadd but directly register the std vector